src/pages/index.js

```diff
@@ -19,6 +19,23 @@ import "../Style/indexStyle.css"
 
 export default class Index extends React.Component {
   render() {
+    let blogPath
+    let gdlFrontendPath
+    const env = process.env.GATSBY_GDL_ENVIRONMENT || "development"
+    switch (env) {
+      case "development":
+        blogPath = "http://localhost:8000/"
+        gdlFrontendPath = "http://localhost:8000/"
+        break
+      case "prod":
+        blogPath = "https://blog.digitallibrary.io/"
+        gdlFrontendPath = "https://digitallibrary.io/"
+        break
+      default:
+        blogPath = `https://${env}.blog.digitallibrary.io/`
```

Comment:
Bra du lagde pr, for jeg ser at jeg også har byttet på her, skal være: `https://blog.${env}.digitallibrary.io/`